lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -81,18 +81,18 @@
 // Browse menu
 // -----------
 .forum-nav-browse-menu-wrapper {
-  border-bottom: 1px solid $gray-l3;
+  border-bottom: 1px solid $forum-color-border;
   background: $gray-l5;
 }
 
 .forum-nav-browse-filter {
   position: relative;
-  border-bottom: 1px solid $gray-l2;
+  border-bottom: 1px solid $forum-color-border;
   padding: ($baseline/4);
 }
 
 .forum-nav-browse-filter .icon {
-  @include font-size(12);
+  font-size: 12px;
```

Comment:
We should use one of our defined size variables, or use rem/em. Also 12px is tiny. 16px should be our minimum. If we absolutely need smaller, 14px.